.github/workflows/pull_request_template.md

```diff
@@ -0,0 +1,23 @@
+# [JIRA Ticket ID](Link to Ticket)
+<!-- Please provide explicit URL link to the corresponding JIRA ticket -->
+
+# Summary
+<!-- Please provide a high level overview of what changes have been made. -->
+
+# Changes in this PR
+<!-- Highlight any high level architecture changes if the summary doesn't already cover the scope. -->
+
+# Test Plan
+<!-- Talk through any unit tests added, and if this is a bug fix, please add repro steps in the event the fix needs to be verified. -->
+
+# Screenshots
```

Comment:
How useful is a screenshots section considering most external PRs are likely to be small bugfixes and such?

Reply:
Ah yeah, this is definitely a more "optional" section. Really it's meant more as a supplement to Test Plan, and I also envisioned folks pasting their command line outputs, JSON blobs, etc. here. More than open to revising the language on this. (Or removing outright)

Reply:
I think changing it to `Screenshots (optional)` would be good, it prompts the idea without requiring it.